cmd/controller-manager/app/controllermanager.go

```diff
@@ -205,6 +217,7 @@ func init() {
 	controllers["applicationFailover"] = startApplicationFailoverController
 	controllers["federatedHorizontalPodAutoscaler"] = startFederatedHorizontalPodAutoscalerController
 	controllers["cronFederatedHorizontalPodAutoscaler"] = startCronFederatedHorizontalPodAutoscalerController
+	controllers["replicasSyncer"] = startReplicasSyncerController
```

Comment:
For the controller name, I would suggest going with `hpaReplicasSyncer`, more clear.

Reply:
Done.